readme.md

```diff
@@ -21,29 +21,42 @@ They are easy to use and you do not need to be an expert to get first meaningful
   - [Analysis Tools](#analysis-tools)
   - [Extraction Tools](#extraction-tools)
   - [Support Tools](#support-tools)
+  - [Misc Tools](#misc-tools)
 - [Hardware Tools](#hardware-tools)
+- [Bluetooth BLE Tools](#bluetooth-ble-tools)
+- [ZigBee Tools](#zigbee-tools)
+- [SDR Tools](#sdr-tools)
+- [RFID NFC Tools](#rfid-nfc-tools)
```

Comment:
Broken ToC item due to broken header formatting

Reply:
Fix on the latest commit.